app/oauth2-server-config/client/admin/views/oauthApp.js

```diff
@@ -83,7 +100,7 @@ Template.oauthApp.events({
 		if (name === '') {
 			return toastr.error(TAPi18n.__('The_application_name_is_required'));
 		}
-		if (redirectUri === '') {
+		if (redirectUri === '' || !checkValidURLList(redirectUri)) {
```

Comment:
should we provide a better error? invalid doesn't mean empty, create a key 'The_redirectUri_is_invalid'